### PR TITLE
Move /x/net/context to context in docker client README

### PR DESCRIPTION
**Pull request by jchorl**
**\- What I did**
Move the golang.org/x/net/context to context to comply with golang 1.7

**\- How I did it**
Made some text changes

**\- How to verify it**
Look

**\- Description for the changelog**
Text change

Move /x/net/context to context in docker client README

Signed-off-by: Josh Chorlton jchorlton@gmail.com

### CODE REVIEW

**Comment by justincormack** [comment]
We are currently still supporting builds with Go 1.6 eg for Linux distros
that build themselves, so I don't think we can do this yet.

**Comment by justincormack** [comment]
Oh for the README only that is fine though.

File: client/README.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ package main
 
 import (
 	"fmt"
+        "context"
```
Comment (justincormack) [inline client/README.md]: The indentation doesnt look right if you view the file - tabs vs spaces?

**Comment by thaJeztah** [comment]
LGTM

ping @justincormack ptal

**Comment by justincormack** [comment]
LGTM. Janky failure unrelated.

**Pull request merged**